Description: Use /etc/default instead of /etc/sysconfig as CONFIGDIR
Index: p910nd/Makefile
===================================================================
--- p910nd.orig/Makefile	2011-02-15 13:09:31.178741585 +0100
+++ p910nd/Makefile	2011-02-15 13:10:57.615777564 +0100
@@ -22,7 +22,7 @@
 MANPAGE = p910nd.8
 INSTALL = install
 BINDIR = /usr/sbin
-CONFIGDIR = /etc/sysconfig
+CONFIGDIR = /etc/default
 SCRIPTDIR = /etc/init.d
 MANDIR = /usr/share/man/man8
 
